Library/Homebrew/extend/os/linux/gnu_tar.rb

```diff
@@ -0,0 +1,14 @@
+module UnpackStrategy
+  class Tar
+    include UnpackStrategy
+
+    using Magic
+
+    private
+
+    def extract_to_dir(unpack_dir, basename:, extra_params:, verbose:)
+      extra_params.value = "--user=0 --group=0 --no-same-owner"
```

Comment:
This should be an array with three separate arguments.

Reply:
ok